network/protocol.go

```diff
@@ -32,6 +33,18 @@ import (
 	"github.com/ethersphere/swarm/state"
 )
 
+var (
+	capabilitiesFlagRetrieve      = []byte{0x00, 0x01} // node retrieves data for itself
+	capabilitiesFlagPush          = []byte{0x00, 0x02} // node pushes own data to network
+	capabilitiesFlagRelayRetrieve = []byte{0x00, 0x10} // node relays retrieve requests for the network
+	capabilitiesFlagRelayPush     = []byte{0x00, 0x20} // node relays push requests for the network
+	capabilitiesFlagStorer        = []byte{0x80, 0x00} // node is part of network storage (sync)
+
+	// temporary presets to emulate the legacy LightNode/full node regime
+	fullCapability  capability
```

Comment:
you're using the term `fullCapability` but above you use `capability` to refer to a certain capability within a certain module but as far as i understand this might infer capabilities over several modules...?

Reply:
The "full" in this context is only about the bzz capabilities. Swap, pss etc will have other distinct capabilities.